### DISCUSSION TITLE
Characters trading

### DISCUSSION THREAD

**Discussion by gettheboom** [chromiecraft/chromiecraft#3290] [Suggestions]
Hi,

Not sure how long this might take, but could you make characters tradeable between players on your website for chromiepoints?
In my opinion, it would only bring more money towards the server wealth, attracting more players to donate

Regards,
Gettheboom

**Comment by aradep** [comment]
This would undermine the efforts of every player who had to level at 1x. If you want to pay to skip the line, go to a p2w server imo.

**Comment by Annamaria-CC** [comment]
Hey Gettheboom,

We have been discussing this within staff and we think that allowing character trading is open for too much abuse and at this point we are not going to implement it.
